lib/Sema/TypeCheckType.cpp

```diff
@@ -595,13 +609,12 @@ Type TypeChecker::resolveTypeInContext(
     if (!foundDC->getDeclaredInterfaceType())
       return ErrorType::get(ctx);
 
-    selfType = resolution.mapTypeIntoContext(
-      foundDC->getDeclaredInterfaceType());
+    selfType =
+        resolution.mapTypeIntoContext(foundDC->getDeclaredInterfaceType());
```

Comment:
There's a lot of re-formatting here which makes the patch harder to review. Is it really necessary?